config/templates/example.yaml.in

```diff
@@ -0,0 +1,44 @@
+# Copyright {{ .Year }} The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# {{ .GeneratedWarning }}
+#
+
+apiVersion: crdb.cockroachlabs.com/v1alpha1
+kind: CrdbCluster
+metadata:
+  name: cockroachdb
+spec:
+  dataStore:
+    pvc:
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: "60Gi"
+        volumeMode: Filesystem
+  resources:
+    requests:
+      cpu: "2"
+      memory: "8Gi"
+    limits:
+      cpu: "2"
+      memory: "8Gi"
+  tlsEnabled: true
+{{- /* .LatestStableCrdbVersion: latest stable version of CockroachDB */}}
+# cockroachDBVersion: {{ .LatestStableCrdbVersion }}
```

Comment:
Do we want this commented out?

Reply:
I don't know, TBH. Cargo culting for now.

Reply:
ack